File: apps/repair/fix_file.py
#!/usr/bin/env python3
"""
CustomerRequest.tsx 파일을 1280번 줄까지만 유지하고 나머지를 삭제하는 스크립트
"""

file_path = 'src/app/components/CustomerRequest.tsx'

# 파일 읽기
with open(file_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# 1280번 줄까지만 유지
lines_to_keep = lines[:1280]

# 파일 덮어쓰기
with open(file_path, 'w', encoding='utf-8') as f:
    f.writelines(lines_to_keep)

print(f"✅ 파일이 성공적으로 정리되었습니다!")
print(f"   유지된 줄 수: {len(lines_to_keep)}")
print(f"   삭제된 줄 수: {len(lines) - len(lines_to_keep)}")

File: repair/fix_file.js
#!/usr/bin/env node
/**
 * CustomerRequest.tsx 파일을 1280번 줄까지만 유지하고 나머지를 삭제하는 스크립트
 */

const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, 'src/app/components/CustomerRequest.tsx');

console.log('📝 파일 정리 시작...');
console.log(`   대상 파일: ${filePath}`);

// 파일 읽기
const content = fs.readFileSync(filePath, 'utf8');
const lines = content.split('\n');

console.log(`   현재 줄 수: ${lines.length}`);

// 1280번 줄까지만 유지
const linesToKeep = lines.slice(0, 1280);

// 파일 덮어쓰기
const newContent = linesToKeep.join('\n');
fs.writeFileSync(filePath, newContent, 'utf8');

console.log('✅ 파일이 성공적으로 정리되었습니다!');
console.log(`   유지된 줄 수: ${linesToKeep.length}`);
console.log(`   삭제된 줄 수: ${lines.length - linesToKeep.length}`);
